lib/Runtime/Language/JavascriptConversion.cpp

```diff
@@ -305,6 +305,20 @@ namespace Js
                 PropertyString * propertyString = (PropertyString *)propName;
                 *propertyRecord = propertyString->GetPropertyRecord();
             }
+            else if (VirtualTableInfo<Js::LiteralStringWithPropertyStringPtr>::HasVirtualTable(propName))
```

Comment:
Possibly, but I thought by having it here we won't really ever end up with these type of strings inside any type handlers.